Conceptpower+Spring/src/main/resources/validatormessages_en_US.properties

```diff
@@ -19,22 +19,24 @@ concept_name.wordnet = Concept name should not be WordNet.
 required.type_name = Concept Type name is mandatory.
 required.type_description = Concept Type description is mandatory.
 required.unique.type_name = Concept Type should be unique.
-#User Registration Validator Messages
-required.username = Username cannot be empty
-required.name = Name of the User cannot be empty
-required.password = Password cannot be empty
-password.short = Password should be at least 4 characters
-required.email = Email cannot be empty
-proper.email = Please give a proper email id
-username.exists = Username already exists. Please Select a Different Username
-proper.username = Username cannot have any special characters or Capital Letters
-proper.name = Name cannot have any special characters. Only Spaces allowed
-#Concept List Creation
-concept_name.required = Concept name is mandatory
-concept_description.required = Concept Description is mandatory
-concept_unique.required = Concept name should be unique
-concept_name.wordnet = Concept name should not be WordNet
-#HomeScreen Validators
-name.required = Word is mandatory field.
-pos.required = POS is mandatory.
-no.searchResults = No search results found.
\ No newline at end of file
+#User Registration Validator Messages
+required.username = Username cannot be empty.
+required.name = Name of the User cannot be empty.
+required.password = Password cannot be empty.
+password.short = Password should be at least 4 characters.
+required.email = Email cannot be empty.
+proper.email = Please give a proper email id.
+username.exists = Username already exists. Please Select a Different Username.
+proper.username = Username cannot have any special characters or Capital Letters.
+proper.name = Name cannot have any special characters. Only Spaces allowed.
+#Concept List Creation
+concept_name.required = Concept name is mandatory.
+concept_description.required = Concept Description is mandatory.
+concept_unique.required = Concept name should be unique.
+concept_name.wordnet = Concept name should not be WordNet.

```

Comment:
This was used in concept list. And there are two occurrences of concept_name.wordnet. Removed one of the occurrence and renamed the other as concept_list_name.wordnet with the message "Concept list name can not be WordNet."